### PR TITLE
Stories for Zero Height

### PR DESCRIPTION
**Pull request by jeremie-lucca**

### CODE REVIEW

**Comment by LuccaIntegration** [comment]
:woman_cook: http://lucca-front.lucca.local/PR-1558

File: stories/scss/cards/cards-footer.stories.ts [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+	return `
+	<div class="card">
+		<div class="card-content">
+			<h2>Titre de la carte</h2>
+			<p>Contenu de la carte</p>
+		</div>
+		<footer class="card-footer">
+			<div class="card-footer-right">
+				<button class="button palette-primary">Confirmer</button>
+				<button class="button mod-link">Annuler</button>
```
Comment (MarcFairbrother) [inline stories/scss/cards/cards-footer.stories.ts:19]:
```suggestion
				<button class="button palette-primary" type="button">Confirmer</button>
				<button class="button mod-link" type="button">Annuler</button>
```

File: stories/scss/empty-state/empty-state-basic.stories.ts [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+export default {
+	title: 'SCSS/EmptyState/Basic',
+} as Meta;
+
+function getTemplate(args: EmptyStateBasicStory): string {
+	return `
+	<section class="emptyState">
+		<h3 class="emptyState-title">Shhh, c'est calme ici</h3>
+		<p class="emptyState-description">Vous pouvez suggérer ici une action à réaliser</p>
+		<button class="button palette-primary size-big">Faire une action</button>
```
Comment (MarcFairbrother) [inline stories/scss/empty-state/empty-state-basic.stories.ts:15]:
```suggestion
		<button class="button palette-primary size-big" type="button">Faire une action</button>
```

**Review by GuillaumeNury — APPROVED** [review]
LGTM

File: stories/scss/gauge/gauge.stories.ts [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+	},
+} as Meta;
+
+function getTemplate(args: GaugeBasicStory): string {
+	const classes = [args.palette].filter(Boolean).join(' ');
+	const thin = args.thin ? `mod-thin` : '';
+	return `
+	<div class="gauge ${classes} ${thin}">
+		<div class="gauge-bar" style="width:33%"></div>
+	</div>
```
Comment (MarcFairbrother) [inline stories/scss/gauge/gauge.stories.ts:31]: We should add an `aria-hidden` and a masked text for a11y reasons.

File: stories/scss/headers/headers-basic.stories.ts [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+export default {
+	title: 'SCSS/Headers/Basic',
+} as Meta;
+
+function getTemplate(args: HeadersBasicStory): string {
+	return `
+	<div class="header">
+		<div class="header-contentLeft">
+	  	<div class="textfield mod-search">
+	  		<input class="textfield-input" type="text" placeholder="Rechercher">
```
Comment (MarcFairbrother) [inline stories/scss/headers/headers-basic.stories.ts:15]: Missing label

File: stories/scss/headers/headers-basic.stories.ts [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+	  	</div>
+		</div>
+		<div class="header-contentRight">
+			<button class="button palette-primary">button</button>
```
Comment (MarcFairbrother) [inline stories/scss/headers/headers-basic.stories.ts:19]:
```suggestion
			<button class="button palette-primary" type="button">button</button>
```

File: stories/scss/headers/headers-navigation.stories.ts [hunk 6]
```diff
@@ -0,0 +1,28 @@
...
+
+export default {
+	title: 'SCSS/Headers/Navigation',
+} as Meta;
+
+function getTemplate(args: HeadersNavigationStory): string {
+	return `
+	<div class="header mod-nav">
+		<div class="header-nav">
+			<a class="header-nav-back"></a>
```
Comment (MarcFairbrother) [inline stories/scss/headers/headers-navigation.stories.ts:14]:
```suggestion
			<a class="header-nav-back" href="#"></a>
```

File: stories/scss/loadings/loadings-basic.stories.ts [hunk 7]
```diff
@@ -0,0 +1,61 @@
...
+	},
+} as Meta;
+
+function getTemplate(args: LoadingsBasicStory): string {
+	const label = args.label ? `Loading…` : '';
+	const block = args.block ? `mod-block` : '';
+	const big = args.big ? `mod-big` : '';
+	const invert = args.invert ? `mod-invert` : '';
+	return `
+	<div class="loading ${block} ${big} ${invert}">${label}</div>
```
Comment (MarcFairbrother) [inline stories/scss/loadings/loadings-basic.stories.ts:42]:
```suggestion
	<div class="loading ${block} ${big} ${invert}" aria-hidden="true">${label}</div>
```

File: stories/scss/progression/progression-basic.stories.ts [hunk 8]
```diff
@@ -0,0 +1,41 @@
...
+	},
+} as Meta;
+
+function getTemplate(args: ProgressionBasicStory): string {
+	const classes = [args.status].filter(Boolean).join(' ');
+	const indeterminate = args.indeterminate ? `mod-indeterminate` : '';
+	return `
+	<div class="progress ${classes} ${indeterminate}">
+		<div class="progress-bar" style="width: 50%"></div>
+	</div>
```
Comment (MarcFairbrother) [inline stories/scss/progression/progression-basic.stories.ts:31]: Missing `aria-hidden` and masked text

File: stories/scss/timelines/timelines-basic.stories.ts [hunk 9]
```diff
@@ -23,23 +23,32 @@ export default {
...
 	const number = args.number ? `mod-number` : '';
 	return `
-	<ol class="timeline ${classes} ${number}">
+	<ol class="timeline ${stateTxt} ${number}">
 		<li class="timeline-step">
 			<span class="timeline-step-title">
 				<a href="#" class="timeline-step-title-action">Previous step</a>
 			</span>
-			<div class="gauge mod-thin">
+			<div class="gauge mod-thin ${stateGauge}">
```
Comment (MarcFairbrother) [inline stories/scss/timelines/timelines-basic.stories.ts:43]:
```suggestion
			<div class="gauge mod-thin ${stateGauge}" aria-hidden="true">
```

**Review by MarcFairbrother — APPROVED** [review]

**Pull request merged**